burn-wgpu/src/template/matmul/blocktiling_2d/continuous.wgsl

```diff
@@ -79,18 +87,21 @@ fn main(
             let lhs_sm_position = thread_offset + load_index;
             let block_row = lhs_sm_position / B_K;
             let block_col = lhs_sm_position % B_K;
-            let lhs_position = offset_lhs + k + block_row * K + block_col;
+            let lhs_position = offset_lhs + block_row * lhs_stride_row + (k + block_col) * lhs_stride_col;
 
             if block_row < B_M {
                 shared_lhs[lhs_sm_position] = lhs[lhs_position];
+            } else {
+                // Patch for mac os bugfix
```

Comment:
It'd be good to mention the name of the bug or behavior for the future maintainers to revert the fix if bug is no longer relevant. Do we have an external reference or internal?